content/documentation/pages/6-recipes/9-cloud-providers/1-gke-regional-clusters.md

```diff
@@ -0,0 +1,407 @@
+mysql   Bound    pvc-24f2acb5-17cd-45e1-8064-34bf602e408f   8Gi        RWO            standard       3m1s
+```
+
+Check the zone the volume is located in:
+
+```
+$ kubectl get pv/pvc-24f2acb5-17cd-45e1-8064-34bf602e408f -o jsonpath='{.metadata.labels.failure-domain\.beta\.kubernetes\.io/zone}'
+us-east1-d
+```
+
+Check the MySQL pod to verify its running and the node allocated to:
+
+```
+$ kubectl get pod/mysql-b94654bd4-9zpt2 -o=custom-columns=NAME:.metadata.name,STATUS:.status.phase,NODE:.spec.nodeName
+NAME                    STATUS    NODE
+mysql-b94654bd4-9zpt2   Running   gke-regional-demo-default-pool-fb3e6608-0rcc
+```
+
+And finally the zone the node resides in can be verified by:
+
+```
+$ kubectl get node gke-regional-demo-default-pool-fb3e6608-0rcc -o jsonpath='{.metadata.labels.failure-domain\.beta\.kubernetes\.io/zone}'
+us-east1-d
+```
+
+### Deploying RabbitMQ
+
+<!--NOTE-->
+
+In this recipe, an instance of RabbitMQ is being deployed to a single zone. Please see the relevant high availablity documentation of the [RabbitMQ](https://www.rabbitmq.com/documentation.html) product for more details. In the event of a zone failure, RabbitMQ may be unavailable.
+
+<!--END_NOTE-->
+
+The deployment of RabbitMQ will consist of 1 replica. The provided manifests do not configure any persistent storage as with MySQL. In order to place RabbitMQ in a specific zone, we can use a simple [nodeSelector](https://kubernetes.io/docs/concepts/scheduling-eviction/assign-pod-node/#nodeselector).
+
+The node in which a the pod gets scheduled to may not be a concern, just that it resides in a specific zone. All nodes in the cluster automatically get labels assigned, one of those representing the zone the node resides which we will utilize.
+
+Make the following modification to the `rabbitmq/rabbitmq-deployment.yaml` file to place RabbitMQ on a node in the `us-east1-b` zone:
+
+```
+    spec:
+      nodeSelector:
+        failure-domain.beta.kubernetes.io/zone: us-east1-b
+```
+
+Deploy the manifests:
+
+```
+$ kubectl create -f rabbitmq/
+deployment.apps/rabbitmq created
+service/rabbitmq created
+```
+
+Get the node the pod is deployed to:
+
+```
+$ kubectl get pod/rabbitmq-6d65f675d9-4vksj -o jsonpath='{.spec.nodeName}'
+gke-regional-demo-default-pool-ea10f422-5f72
+```
+
+Get the zone the node resides in:
+
+```
+$ kubectl get node gke-regional-demo-default-pool-ea10f422-5f72 -o jsonpath='{.metadata.labels.failure-domain\.beta\.kubernetes\.io/zone}'
+us-east1-b
+```
+
+### Deploying Skipper
+
+The deployment of Skipper will consist of 3 replicas. No persistent storage is needed and its desired to have a replica running in each zone. Multiple replica's should not reside in the same zone as another. One way to do this would be using [Pod Anti Affinity](https://kubernetes.io/docs/concepts/scheduling-eviction/assign-pod-node/#inter-pod-affinity-and-anti-affinity).
```

Comment:
```suggestion
The deployment of Skipper will consist of 3 replicas. No persistent storage is needed and its desired to have a replica running in each zone. Multiple replicas should not reside in the same zone as another. One way to do this would be using [Pod Anti Affinity](https://kubernetes.io/docs/concepts/scheduling-eviction/assign-pod-node/#inter-pod-affinity-and-anti-affinity).
```